spec/spec_helper.rb

```diff
@@ -26,11 +26,18 @@ def s3
 
 RSpec.configure do |config|
   def converter
-    @converter ||= if !StaticData::JWT_ENABLE
+    @converter ||= if PretestsCheck.jwt_enable?
+                     if StaticData.documentserver_jwt_exist? && StaticData.documentserver_jwt_empty?
```

Comment:
Maybe cobine output of both of those methods in 

```
def jwt_in_env?
  StaticData.documentserver_jwt_exist? && !StaticData.documentserver_jwt_empty?
end
```

and rename `documentserver_jwt_exist` `jwt_in_env?` and `documentserver_jwt_empty` to `jwt_in_env_empty?`

And rename `jwt_data_exist` to `jwt_in_config_file?`